internal/pkg/feature/cmd/feature/main.go

```diff
@@ -0,0 +1,275 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"go/format"
+	"io/ioutil"
+	"os"
+	"strings"
+	"text/template"
+
+	"gopkg.in/yaml.v2"
+)
+
+const tmpl = `// Code generated by the feature package; DO NOT EDIT.
+
+package {{.PackageName}}
+
+import (
+	"context"
+
+	"github.com/influxdata/flux/internal/pkg/feature"
+)
+
+type (
+	Flag       = feature.Flag
+	Flagger    = feature.Flagger
+	StringFlag = feature.StringFlag
+	FloatFlag  = feature.FloatFlag
+	IntFlag    = feature.IntFlag
+	BoolFlag   = feature.BoolFlag
+)
+
+{{ range $_, $flag := .Flags }}
+var {{ $flag.Key }} = feature.{{ $flag.Default | maker }}(
+	{{ $flag.Name | quote }},
+	{{ $flag.Key | quote }},
+	{{ $flag.Contact | quote }},
+	{{ $flag.Default | conditionalQuote }},
+)
+
+// {{ $flag.Name | replace " " "_" | camelcase }} - {{ $flag.Description }}
+func {{ $flag.Name | replace " " "_" | camelcase }}() {{ $flag.Default | flagType }} {
+	return {{ $flag.Key }}
+}
+{{ end }}
+
+// Inject will inject the Flagger into the context.
+func Inject(ctx context.Context, flagger Flagger) context.Context {
+	return feature.Inject(ctx, flagger)
+}
+
+var all = []Flag{
+{{ range $_, $flag := .Flags }}	{{ $flag.Key }},
+{{ end }}}
+
+var byKey = map[string]Flag{
+{{ range $_, $flag := .Flags }}	{{ $flag.Key | quote }}: {{ $flag.Key }},
+{{ end }}}
+
+// Flags returns all feature flags.
+func Flags() []Flag {
+	return all
+}
+
+// ByKey returns the Flag corresponding to the given key.
+func ByKey(k string) (Flag, bool) {
+	v, found := byKey[k]
+	return v, found
+}
+`
+
+type flagConfig struct {
+	Name        string
+	Description string
+	Key         string
+	Default     interface{}
+	Contact     string
+}
+
+func (f flagConfig) Valid() error {
+	var problems []string
+	if f.Key == "" {
+		problems = append(problems, "missing key")
```

Comment:
lol...definitely a semantic variable name.

Reply:
This code was copied from the influxdb version.